src/core.js

```diff
@@ -66,7 +67,12 @@ function callPenthouse(document, options) {
 
   return sizes.map(({width, height}) => {
     debug('Call penthouse with:', {...config, width, height});
-    return () => penthouse({...config, width, height});
+    return async () => {
+      // Resolves some weird bug in penthouse when multiple dimensions.
+      // Needs further investigation 
+      await delay(10);
```

Comment:
would love to have more info in this - to help you debug, and fix if there's a bug in penthouse!

Reply:
I’m on vacation at the moment. I’ll get in touch with you next week. Thanks 😊

Reply:
In case the timing is right, I suspect I just fixed the bug you came across, here:
https://github.com/pocketjoso/penthouse/pull/279

It's not yet published into `latest` for Penthouse, but should be in a few days. There's a release candidate that has the fix, if you want to test it sooner:
`penthouse@1.10.1-rc`.

Reply:
\o/ I'll give it a try. Thanks @pocketjoso